app/views/catalog/_st_form.html.haml

```diff
@@ -5,13 +5,14 @@
   - # show form if Catalog bundle is being added or if it's an edit of existing catalog item, else force user to select type
   - if @edit[:new][:service_type] == "atomic" && !@edit[:wf] && !@edit[:new][:st_prov_type]
     %h3=_('New Catalog Item')
```

Comment:
@PanSpagetka You can remove this title. It's redundant since it follows "Adding a new Service Catalog Item"